compiler/src/iree/compiler/Codegen/Interfaces/BufferizationInterfaces.cpp

```diff
@@ -640,6 +779,22 @@ void registerBufferizationInterfaces(DialectRegistry &registry) {
         IREE::TensorExt::DispatchTensorStoreOp::attachInterface<
             DispatchTensorStoreOpSubsetInsertionInterface>(*ctx);
       });
+  registry.addExtension(+[](MLIRContext *ctx,
+                            IREE::Codegen::IREECodegenDialect *dialect) {
+    // LoadFromMemrefOp
+    IREE::Codegen::LoadFromMemrefOp::attachInterface<LoadFromMemrefOpInterface>(
+        *ctx);
+    IREE::Codegen::LoadFromMemrefOp::attachInterface<
+        LoadFromMemrefOpSubsetInterface>(*ctx);
```

Comment:
I think you can just collapse them into a single statement and remove the `LoadFromMemrefOp` comment.

```
    IREE::Codegen::LoadFromMemrefOp::attachInterface<
        LoadFromMemrefOpInterface,
        LoadFromMemrefOpSubsetInterface>(*ctx);
```